Description: Adjous paths for .pybuild directory, to fix a test failure
Author: Gianfranco Costamagna <locutusofborg@debian.org>
Last-Update: 2025-07-23

--- borgbackup2-2.0.0b17.orig/src/borg/testsuite/shell_completions_test.py
+++ borgbackup2-2.0.0b17/src/borg/testsuite/shell_completions_test.py
@@ -3,7 +3,7 @@ import os
 
 import pytest
 
-SHELL_COMPLETIONS_DIR = Path(__file__).parent / ".." / ".." / ".." / "scripts" / "shell_completions"
+SHELL_COMPLETIONS_DIR = Path(__file__).parent / ".." / ".." / ".." / ".." / ".." / "scripts" / "shell_completions"
 
 
 def test_bash_completion_is_valid():
